### PR TITLE
Rebrand DCS

### PR DESCRIPTION
**Pull request by Lulalaby**
[T2](https://bugs.aitsys.dev/T2)

### CODE REVIEW

**Comment by OoLunar** [comment]
Simply renaming packages and namespaces isn't enough. We should point out that this is a fork of DSP, and completely redo the README

**Comment by Lulalaby** [comment]
Yes I know, but that's not my part.
@JMLutra is mainly assigned for it.
I just do the preparations.

We do a full rebrand of logos, texts, informations, names, namespaces and so on

File: README.md [hunk 1]
```diff
@@ -29,16 +29,16 @@ We want this lib always up-to-date. So the API version is always the newest, in
 ## Installing
 You can install the library from following source:
 
-The latest release is always available on [NuGet](https://nuget.dspng.aitsys.dev).
+The latest release is always available on [NuGet](https://nuget.dcs.aitsys.dev).
 
 ## Documentation
-The documentation for the latest stable version is available at [docs.dspng.aitsys.dev](https://docs.dspng.aitsys.dev).
+The documentation for the latest stable version is available at [docs.dcs.aitsys.dev](https://docs.dcs.aitsys.dev).
```
Comment (OoLunar) [inline README.md:35]: Broken url. May want to update DNS records

File: README.md [hunk 2]
```diff
@@ -50,14 +50,14 @@ Feel free to re-use code in DSharpPlus.
 ## NuGet
 Package|NuGet
 |--|--|
-DSharpPlusNextGen|[![NuGet](https://img.shields.io/nuget/vpre/DSharpPlusNextGen.svg?label=)](https://nuget.dspng.aitsys.dev/DSharpPlusNextGen)
+DisCatSharp|[![NuGet](https://img.shields.io/nuget/vpre/DSharpPlusNextGen.svg?label=)](https://nuget.dspng.aitsys.dev/DSharpPlusNextGen)
 DSharpPlusNextGen.CommandsNext|[![NuGet](https://img.shields.io/nuget/vpre/DSharpPlusNextGen.CommandsNext.svg?label=)](https://nuget.dspng.aitsys.dev/DSharpPlusNextGen.CommandsNext)
```
Comment (OoLunar) [inline README.md:54]: CommandsNext wasn't renamed? Is this because it's going to be chopped or was it missed on accident?

**Review by OoLunar — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (Lulalaby) [reply README.md:54]: 2nd.

File: README.md (hunk 1 above)
Comment (Lulalaby) [reply README.md:35]: yeah I'm doing the update on merge

File: README.md (hunk 2 above)
Comment (Lulalaby) [reply README.md]: fixed

File: README.md [hunk 3]
```diff
@@ -1,5 +1,5 @@
-# DSharpPlusNextGen ![GitHub](https://img.shields.io/github/license/Aiko-IT-Systems/DSharpPlusNextGen?label=License) ![Sponsors](https://img.shields.io/github/sponsors/Lulalaby?label=Sponsors)  ![Discord Server](https://img.shields.io/discord/858089281214087179.svg?label=Discord)
-Discord Bot Library written in C# for .NET - based off [DSharpPlus](https://github.com/DSharpPlus/DSharpPlus).
+# DisCatSharp ![GitHub](https://img.shields.io/github/license/Aiko-IT-Systems/DSharpPlusNextGen?label=License) ![Sponsors](https://img.shields.io/github/sponsors/Lulalaby?label=Sponsors)  ![Discord Server](https://img.shields.io/discord/858089281214087179.svg?label=Discord)
+Discord Bot Library written in C# for .NET.
```
Comment (JMLutra) [inline README.md:2]: I would add something like "originally based off DSharpPlus"

Comment (Lulalaby) [reply README.md:2]: yes

Comment (Lulalaby) [reply README.md:2]: @OoLunar what do u think

Comment (OoLunar) [reply README.md:2]: I think the "Written in C# for .NET" part is a bit redundant, considering .NET is an ecosystem that I believe Microsoft is trying to turn into one general thing, if not phase out entirely. Perhaps something like "Originally based off of DSharpPlus, DisCatSharps aims to fix DSP's internal mistakes, without any internal drama."

Comment (Lulalaby) [reply README.md:2]: yeah seems right

**Review by JMLutra — APPROVED** [review]

**Pull request merged**